shared/login/register/paper-key/index.render.native.js

```diff
@@ -13,8 +13,10 @@ class PaperKeyRender extends Component<void, Props, void> {
           style={stylesContainer}
           onBack={this.props.onBack}>
           <Text type='Header' style={stylesHeader}>Type in your paper key:</Text>
-          <Icon type='icon-paper-key-48' style={stylesIcon} />
+          <Icon type='icon-paper-key-48' />
           <Input
+            multiline={true}
+            rows={3}
             autoCorrect={true}
```

Comment:
autocorrect should be false.

Maybe type should be passwordVisible? I don't know if that will mess with multiline.